packages/e2e-tests/specs/editor/various/writing-flow.test.js

```diff
@@ -550,36 +550,6 @@ describe( 'Writing Flow', () => {
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
 
-	it( 'should not have a dead zone between blocks (upper)', async () => {
```

Comment:
Why did you remove this behaviour?

Reply:
I believe it's related to this https://github.com/WordPress/gutenberg/issues/28682#issuecomment-775741614

Basically, I just though that it doesn't make sense to focus the previous or the next block depending on a a small gap (the margin is very small) and just chose to focus the next one every time. Thought after discussing on that issue, I think focusing the end of the previous block or the start of the next block would be better.